packages/configuration-all/index.json

```diff
@@ -0,0 +1,78 @@
+{
+    "connector": "puppeteer",
+    "formatters": [
+        "html",
+        "summary"
+    ],
+    "hints": {
+        "amp-validator": "error",
+        "apple-touch-icons": "error",
+        "axe/aria": "error",
+        "axe/forms": "error",
+        "axe/color": "error",
+        "axe/keyboard": "error",
+        "axe/language": "error",
+        "axe/name-role-value": "error",
+        "axe/other": "error",
+        "axe/parsing": "error",
+        "axe/semantics": "error",
+        "axe/sensory-and-visual-cues": "error",
+        "axe/tables": "error",
+        "axe/structure": "error",
+        "axe/text-alternatives": "error",
+        "axe/time-and-media": "error",
+        "babel-config/is-valid": "error",
+        "button-type": "error",
+        "compat-api/css": "error",
+        "compat-api/html": "error",
+        "content-type": "error",
+        "create-element-svg": "error",
+        "css-prefix-order": "error",
+        "disown-opener": "error",
+        "doctype": "error",
+        "highest-available-document-mode": "error",
+        "html-checker": "error",
+        "http-cache": "error",
+        "http-compression": "error",
+        "https-only": "error",
+        "image-optimization-cloudinary": "error",
+        "manifest-app-name": "error",
+        "manifest-exists": "error",
+        "manifest-file-extension": "error",
+        "manifest-is-valid": "error",
+        "meta-charset-utf-8": "error",
+        "meta-theme-color": "error",
+        "meta-viewport": "error",
+        "minified-js": "error",
+        "no-bom": "error",
+        "no-broken-links": "error",
+        "no-disallowed-headers": "error",
+        "no-friendly-error-pages": "error",
+        "no-html-only-headers": "error",
+        "no-http-redirects": "error",
+        "no-p3p": "error",
+        "no-protocol-relative-urls": "error",
+        "no-vulnerable-javascript-libraries": "error",
+        "performance-budget": "error",
+        "sri": "error",
+        "ssllabs": "error",
+        "strict-transport-security": "error",
+        "stylesheet-limits": "error",
+        "typescript-config/consistent-casing": "error",
+        "typescript-config/import-helpers": "error",
+        "typescript-config/is-valid": "error",
+        "typescript-config/strict": "error",
+        "typescript-config/no-comments": "error",
+        "typescript-config/target": "error",
+        "validate-set-cookie-header": "error",
+        "webpack-config/config-exists": "error",
+        "webpack-config/is-installed": "error",
+        "webpack-config/is-valid": "error",
+        "webpack-config/module-esnext-typescript": "error",
+        "webpack-config/modules-false-babel": "error",
+        "webpack-config/no-devtool-in-prod": "error",
+        "x-content-type-options": "error"
+    },
+    "hintsTimeout": 120000,
+    "parsers": []
```

Comment:
I missed this before, but I think we need to include all the parsers here if we want this configuration to fully enable all the hints. Otherwise those that are waiting for a parse event from a specific parser will never run.

Reply:
ups, you are right!

Reply:
Also, random thought about future behavior: should we even specify parsers via `.hintrc` anymore? It just seems like a way to have hints not work by forgetting them. I'd prefer having hints specify which parser(s) they need and having those automatically pulled in based on which hints are enabled.

@sarvaje If that sounds reasonable to you, I'll file this as an issue so we can discuss with the broader team later. What do you think?

Reply:
Sounds reasonable, but for now, we need to fill that list :)